final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import re
+import xml.etree.ElementTree as ET
+import requests
+import json
+from datetime import datetime
+
+import RssReaderExceptions as rre
```

Comment:
It seems that you forgot to commit this file

Reply:
Yes, but now I've decided to use the exception class from requests.exceptions instead